pythonsdk/src/main/python/vantiqservicesdk.py

```diff
@@ -187,33 +187,32 @@ def __complete_request(completed_task):
                     task.cancel()
 
     async def __process_message(self, request: dict) -> Any:
+        # Get the procedure name and parameters
         self._logger.debug('Request was: %s', request)
-        procedure_name: str = 'unknown'
-        try:
-            # Get the procedure name and parameters
-            procedure_name = request.get("procName")
-            params = request.get("params")
-
-            # In the future, it should be set by an explicit value isSystemRequest. For compatibility reasons, we
-            # must accept appending "__system" to the requestId as equivalent
-            is_system_request = request.get("isSystemRequest", False) or request.get("requestId").endswith("__system")
-
-            # Invoke the procedure and store the result
-            request_timer = self.__get_resource_metric(self._resources_executions, procedure_name)
-            with request_timer.time():
-                result = await self.__invoke(procedure_name, params, is_system_request)
-                return result
-        except Exception as e:
-            # Log and record error
-            self._logger.debug(f"Error invoking procedure {procedure_name}", exc_info=e)
-            self.__get_resource_metric(self._failed_requests, procedure_name).inc()
-            raise e
-    
+        procedure_name: str = request.get("procName", "unknown")
+        params = request.get("params")
+
+        # In the future, it should be set by an explicit value isSystemRequest. For compatibility reasons, we
+        # must accept appending "__system" to the requestId as equivalent
+        is_system_request = request.get("isSystemRequest", False) or request.get("requestId").endswith("__system")
+
+        # Invoke the procedure and store the result
+        request_timer = self.__get_resource_metric(self._resources_executions, procedure_name)
+        with request_timer.time():
+            result = await self.__invoke(procedure_name, params, is_system_request)
+            return result
+
     async def __fire_forget(self, websocket: WebSocket, request: dict) -> None:
+        # Send a response to indicate that the request was received
         response = self.encode_response({"requestId": request.get("requestId"), "isEOF": True})
         await websocket.send({"type": "websocket.send", "bytes": response})
-        await self.__process_message(request)
-    
+
+        # Process the message and ignore any exceptions
+        try:
+            await self.__process_message(request)
+        except Exception as e:
+            self._handle_exception(e, request, None)
```

Comment:
Doesn't return anything, but does let us log any errors the occur in this pathway.